### DISCUSSION TITLE
In Teleport Connect -- show recently connect histories

### DISCUSSION THREAD

**Discussion by samngmco** [gravitational/teleport#39651] [Ideas]
Often times, I find myself logging in and out of Teleport repeatedly, and when I need to log in again, I have to type in the full domain name, which is quite long. It would be helpful if there was an option to display the last 5-10 recently used URLs so that I can quickly reconnect to them without having to manually type the entire domain each time.

**Comment by webvictim** [comment]
cc @ravicious

**Comment by ravicious** [comment]
I've been meaning to open an issue for that for the past week or so. One thing I'd like to ask: when you log out and then back in, do you do this to update your current user cert with a fresh role set, or do you do this to log in as another user to the same cluster?

Edit: In recent versions of Connect, next to the button to log out there's a button to refresh the session by logging in again.
